### ISSUE TITLE
Help with Config

### ISSUE THREAD

**Issue by hesspoint** [BKeyport/MMM-Countdown2#2]
Hi,

cannot get it to work. The mirror still starts but the countdown is not showing up where it is supposed to. This is my config. 

```
{
            module: 'MMM-CountDown2',
            config: {
             	position: "top_center",
             	event: "X-Mas",
             	date: 2022-12-24,
             	showHours: false,
             	showMinutes: false,
             	showSeconds: false,
             	daysLabel: "days"
            }
 },
```

Thanks!

**Comment by BKeyport** [comment]
I'm sorry, I'm unable to fix, as I'm not sure what's going on.

At this point, I'm going to consider the module dead, as it's not working
here either.

Please remove and try something else. I no longer use the module.

**Comment by KidnappedByPirates** [comment]
Hi. I was struggling with this also. I got it to work. The problem appears to be the directory name. Everything has to match.

The directory when I cloned it was MMM-Countdown2. It needs to be MMM-CountDown2 (with the upper case D). It needs to match what is in the MMM-CountDown2.js file (Module.register("MMM-CountDown2"). And I am pretty sure the name of the .js file has to match also (which it does).

I didn't look at the details as to what is different with this from the original MMM-CountDown. I found this one first, so I figured I would try it out.
